rapid7_insightvm/plugin.spec.yaml

```
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: rapid7_insightvm
 title: Rapid7 InsightVM
 description: InsightVM is a powerful vulnerability management tool which finds, prioritizes, and remediates vulnerabilities. This plugin can get top remediations, scan results and start scans
-version: 3.6.0
+version: 3.6.1
```

```suggestion
version: 4.0.0
```
title changes require a full version bump.